### PR TITLE
Added acceptance for set private environment variable flag

### PR DESCRIPTION
**Pull request by dmvieira**
server api now accept private environment variables

### CODE REVIEW

File: api/app_test.go [hunk 1]
```diff
@@ -1357,6 +1357,71 @@ func (s *S) TestSetEnvHandlerShouldSetAPublicEnvironmentVariableInTheApp(c *chec
...
+
+func (s *S) TestSetEnvHandlerShouldSetADoublePrivateEnvironmentVariableInTheApp(c *check.C) {
+	a := app.App{Name: "black-dog", Platform: "zend", Teams: []string{s.team.Name}}
+	err := app.CreateApp(&a, s.user)
+	c.Assert(err, check.IsNil)
+	defer s.deleteApp(&a)
+	url := fmt.Sprintf("/apps/%s/env?:app=%s&private=1", a.Name, a.Name)
+	request, err := http.NewRequest("POST", url, strings.NewReader(`{"DATABASE_HOST":"localhost"}`))
+	c.Assert(err, check.IsNil)
+	request, err = http.NewRequest("POST", url, strings.NewReader(`{"DATABASE_HOST":"127.0.0.1"}`))
```
Comment (andrewsmedina) [inline api/app_test.go]: I believe that this test is wrong. You are overriding the first request by the second without call `setEnv` with the first request. So, this test only set the last environ. To fix it you should call `setEnv` twice, once by request.

File: api/app_test.go [hunk 2]
```diff
@@ -1383,7 +1453,7 @@ func (s *S) TestSetEnvHandlerShouldSetMultipleEnvironmentVariablesInTheApp(c *ch
 	action := rectest.Action{
 		Action: "set-env",
 		User:   s.user.Email,
-		Extra:  []interface{}{"app=" + a.Name, envs},
+		Extra:  []interface{}{"app=" + a.Name, envs, "private=false"},
```
Comment (andrewsmedina) [inline api/app_test.go]: I believe that you should add a code to test this new `setEnvsToApp` behaviour.

**Comment by andrewsmedina** [comment]
@dmvieira thank you!

**Comment by dmvieira** [comment]
Thank you too ;)
Em 05/04/2015 15:50, "Andrews Medina" <notifications@github.com> escreveu:

**Pull request merged**